solutions/day-17/level2/src/components/Experiences.js

```diff
@@ -0,0 +1,19 @@
+import { profileData } from "../data/data";
+import "../styles/Experiences.css";
+
+const Experiences = () => {
+  return (
+    <div className="experiences-content">
+      <div className="experience-title">Experiences</div>
+      {profileData.experiences.map((experience) => (
+        <div className="company-row">
+          <div className="company">{experience.company}</div>
```

Comment:
you can apply optional chaining on the `map()` method `profileData.experiences?.map()` so if the object is null or undefined, instead of throwing an error, it will return undefined.

Reply:
Updated. Thanks.